### PR TITLE
`subscription.list` | Make filter on status explicit

### PR DESCRIPTION
**Pull request by tiagobastos**

### CODE REVIEW

**Review by gdegirmenci — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -3304,6 +3304,10 @@ Download a credit note in a specific format.
             + invoice_id: `60f03745-4289-4ae7-8c74-d797860bc887` (string, optional) - Find subscriptions that generated the given invoice
             + deal_id: `fddf42f9-bda1-0c49-a22b-0bf147cd1e3f` (string, optional) - Filter on subscriptions created from a deal
             + customer (Customer)
+            + status (array[enum], optional) - Filters on status
+                + Members
+                    + active - Filters on active subscriptions
+                    + deactivated - Filters on archived subscriptions
```
Comment (joaoinacio) [inline apiary.apib]: 🤔 archived? deactivated? inactive?

Comment (mrtus) [reply apiary.apib]: I don't think it is archived, deactivated sounds more correct 👌

**Review by gdegirmenci — APPROVED** [review]

**Review by joaoinacio — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -3304,6 +3304,10 @@ Download a credit note in a specific format.
             + invoice_id: `60f03745-4289-4ae7-8c74-d797860bc887` (string, optional) - Find subscriptions that generated the given invoice
             + deal_id: `fddf42f9-bda1-0c49-a22b-0bf147cd1e3f` (string, optional) - Filter on subscriptions created from a deal
             + customer (Customer)
+            + status (array[enum], optional) - Filters on status
```
Comment (JordyDeGroeve) [inline apiary.apib:3307]: Q: Do we really need the description? It does not add anything

**Review by JordyDeGroeve — APPROVED** [review]

File: apiary.apib (hunk 2 above)
Comment (tiagobastos) [reply apiary.apib:3307]: For consistency with the previous filter's description, I would leave them be.

**Review by Jdww — APPROVED** [review]

**Comment by tiagobastos** [comment]
While MC we noticed a small inconsistency in "status" _members_ in this and other endpoints. 
Properties with `array[enum]` are adding `null` to the list of allowed members while that is not correct.

Fixing outside of this PR.

**Review by rainbow-alex — APPROVED** [review]

**Pull request merged**